src/WCSimDetectorConfigs.cc

```diff
@@ -29,13 +29,13 @@ void WCSimDetectorConstruction::SetSuperKGeometry()
   WCPMTRadius = PMT->GetRadius();
   WCIDDiameter          = 33.6815*m; //16.900*2*cos(2*pi*rad/75)*m; //inner detector diameter
   WCIDHeight            = 36.200*m; //"" "" height
-  WCBarrelPMTOffset     = 0.0715*m; //offset from vertical
+  WCBarrelPMTOffset     = WCPMTRadius; //offset from vertical
```

Comment:
For most of the detector configurations, setting the WCBarrelPMTOffset = WCPMTRadius (or even to the expose height of the PMT?) and WCCapEdgeLimit = WCIDDiameter/2.0 - WCPMTRadius is a good approximation. But for this SK configuration, I think these numbers were taken from the SK NIM paper and are supposed to represent what the actual detector looks like. I just did a small test where I set theWCBarrelPMTOffset = 0.0715*m and  WCCapEdgeLimit = 16.9*m in the SetSuperKGeometry() and there didn't seem to be any overlap warnings, so perhaps it's best to leave these values as they were.